backend/server/src/db.rs

```diff
@@ -37,12 +37,31 @@ pub async fn create_pool(config: &config::Db) -> Result<Pool> {
         .context("failed to get DB connection")?;
     connection.execute("SELECT 1", &[]).await
         .context("failed to execute DB test query")?;
-    let n_roots = connection.execute("SELECT * from realms where id = 0", &[]).await
-        .context("failed to check")?;
-    if n_roots < 1 {
-        bail!("no root realm found");
-    } else if n_roots > 1 {
-        bail!("more than one root realm found");
+
+    // ensure database version exists
+    info!("Creating table version");
+    connection.execute(
+        "create table if not exists version (version int not null)", &[]).await
+        .context("failed to schema for table 'realms'")?;
+    if connection.query("SELECT count(*) < 1 from version", &[]).await?[0].get::<_, bool>(0) {
+        info!("Setting database version");
+        connection.execute("insert into version (version) values (0)", &[]).await
```

Comment:
Certainly newline before `.await`, potentially also before `.execute`. Rust's style is rather strict here: if you split an expression/statement over multiple lines, each "item" has to be in its own line. Here, the "item" is one segment of the call chain. But function arguments for example. Either you go into one line or, if it gets too long, you go like this:

```rust
fn foo(
    a: i32,
    b: String,
    long_long_argument_tralalalaalal: bool,
)
```

But you never combine `a` and `b` into one line but let `long_long_argument_tralalalaalal` have its own line.